src/maintainer/knowledge_base.rst

```diff
@@ -1518,3 +1518,61 @@ The tl;dr here is that conda sorts as follows:
 So make sure that you **tag** your package in such a way that the package name
 that conda-build spits out will sort the package uploaded with an ``rc`` label
 higher than the package uploaded with the ``dev`` label.
+
+Perl packages
+=============
+
+Perl has three standard install locations: **core**, **vendor**, and **site**. Here, **core** is used only for the perl itself, **vendor** is used for installing any other conda-forge packages, and **site** is used for the user to install things locally from sources other than conda-forge.
+
+The most commonly used build system for `perl packages <https://github.com/conda-forge/perl-file-which-feedstock>`__ is ``ExtUtils::MakeMaker``. For the archetypical packaging of packages of this form, you can have a look at `perl-file-which recipe <https://github.com/conda-forge/perl-file-which-feedstock/blob/main/recipe/meta.yaml>`__.
```

Comment:
hyphen needed between commonly and used

Also, do you mean "typical" here, or "archetypal"?

Reply:
If it's typical, I'd recommend removing the "the". "For typical packaging of packages of this form..."

Reply:
Thanks for the review Katherine. Actually, I just used the word that was mentioned in the description of the issue, and I didn't know what exactly it meant! ':D
But now that I checked, I think we should just use "typical" as the "archetypal" isn't a very commonly used word so some people might not get it in the first read.